src/integrationtests/java/util/IntegrationTestBase.java

```diff
@@ -0,0 +1,205 @@
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+       http://www.apache.org/licenses/LICENSE-2.0
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+package util;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+
+import javax.annotation.PostConstruct;
+
+import org.apache.commons.io.FileUtils;
+import org.json.JSONException;
+import org.skyscreamer.jsonassert.JSONAssert;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.test.context.support.AbstractTestExecutionListener;
+
+import com.ericsson.ei.flowtests.FlowTestConfigs;
+import com.ericsson.ei.handlers.ObjectHandler;
+import com.ericsson.ei.mongodbhandler.MongoDBHandler;
+import com.ericsson.ei.rmqhandler.RmqHandler;
+import com.ericsson.ei.rules.RulesHandler;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.mongodb.MongoClient;
+import com.mongodb.client.MongoCollection;
+import com.mongodb.client.MongoDatabase;
+
+/**
+ * @author eedleri
+ *
+ */
+public abstract class IntegrationTestBase extends AbstractTestExecutionListener {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(IntegrationTestBase.class);
+    protected static final String MAILHOG_DATABASE_NAME = "mailhog";
+    private static final String EIFFEL_DATABASE_NAME = "eiffel";
+    private static final String EIFFEL_INTELLIGENCE_DATABASE_NAME = "eiffel_intelligence";
+
+    @Autowired
+    private RmqHandler rmqHandler;
+
+    @Autowired
+    public ObjectHandler objectHandler;
+
+    @Autowired
+    private RulesHandler rulesHandler;
+
+    @Autowired
+    private MongoDBHandler mongoDBHandler;
+
+    @Value("${spring.data.mongodb.database}")
+    private String database;
+
+    @Value("${event_object_map.collection.name}")
+    private String event_map;
+
+    private static ObjectMapper objectMapper = new ObjectMapper();
+
+    private static HashMap<String, FlowTestConfigs> configsMap = new HashMap<String, FlowTestConfigs>();
+
+    @PostConstruct
+    public void init() {
+        cleanDatabases();
+    }
+
+    private void cleanDatabases(){
+        mongoDBHandler.dropDatabase(EIFFEL_DATABASE_NAME);
+        mongoDBHandler.dropDatabase(EIFFEL_INTELLIGENCE_DATABASE_NAME);
+        mongoDBHandler.dropDatabase(MAILHOG_DATABASE_NAME);
+    }
+
+    // setFirstEventWaitTime: variable to set the wait time after publishing the
```

Comment:
multi line comments should be 
/*
*
*/

Reply:
I wanted to do that, people said that there are problems writing it like that in java and said that // in each line is better. Except when Java docing.